### PR TITLE
Deps update 10 2024

### PR DESCRIPTION
**Pull request by mdabek-nvidia**
## Category:

**Other**

## Description:

Dependencies update:
- Google Benchmark to v1.9.0
- DLPack to v1.0
- FFTS to of the tree 09/2024
- GoogleTest to v1.15.2
- pybind11 to v2.13.6
- RapidJSON to top of the tree 09/2024

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: DALI-4068

### CODE REVIEW

File: third_party/README.rst [hunk 1]
```diff
@@ -15,15 +15,15 @@ This part of the repository contains extra dependencies required to build DALI,
...
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
-| `FFTS <https://github.com/JanuszL/ffts>`_                        | `Custom fork top-of-tree (Jan 23, 2020) <https://github.com/JanuszL/ffts/tree/c9a9f61a60505751cac385ed062ce2720bdf07d4>`_ | `BSD 3-Clause License <https://github.com/JanuszL/ffts/blob/master/COPYRIGHT>`_                                   |
+| `FFTS <https://github.com/JanuszL/ffts>`_                        | `Custom fork top-of-tree (Sep 6, 2024) <https://github.com/JanuszL/ffts/tree/95489ebcd6fc136c6a76f50f57d43e6072e2bd38>`_  | `BSD 3-Clause License <https://github.com/JanuszL/ffts/blob/master/COPYRIGHT>`_                                   |
```
Comment (JanuszL) [inline third_party/README.rst:20]: Good idea. I forgot that I updated my fork to support more recent gcc compiler available ion manylinux_2_28 image - due to pending https://github.com/NVIDIA/DALI/pull/5608.

**Review by JanuszL — APPROVED** [review]

File: dali/python/bundle-wheel.sh [hunk 2]
```diff
@@ -121,6 +121,22 @@ DEPS_LIST=(
...
+    "${DEPS_PATH}/lib/libaws-c-common.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-c-common.so.1"
+    "${DEPS_PATH}/lib/libaws-c-sdkutils.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-c-io.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-c-cal.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-c-compression.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-c-http.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-c-auth.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-checksums.so.1.0.0"
+    "${DEPS_PATH}/lib/libaws-cpp-sdk-s3.so"
```
Comment (stiepan) [inline dali/python/bundle-wheel.sh:136]: It's already there at L123, isn't it?

**Comment by stiepan** [comment]
For conda, [we have separate recipes](https://github.com/NVIDIA/DALI/tree/main/conda/third_party) for a few of the deps (e.g. ffmpeg) and updating one of those in DALI_deps usually requires upding the recipe too. There's jpeg-turbo among the custom recipes, but, looking at the build log, I am not sure it's actually used.

**Comment by JanuszL** [comment]
> There's jpeg-turbo among the custom recipes
It is not used there anymore.

File: dali/python/bundle-wheel.sh (hunk 2 above)
Comment (mdabek-nvidia) [reply dali/python/bundle-wheel.sh:136]: Good catch - copy paste error ;)

**Comment by mdabek-nvidia** [comment]
!build

**Review by stiepan — APPROVED** [review]

**Pull request merged**